comp/metadata/inventorychecks/inventorychecksimpl/inventorychecks.go

```diff
@@ -197,9 +207,43 @@ func (ic *inventorychecksImpl) getPayload() marshaler.JSONMarshaler {
 		}
 	}
 
+	logsMetadata := make(map[string][]metadata)
+	if sources, isSet := ic.sources.Get(); isSet {
+		if sources != nil {
+			for _, logSource := range sources.GetSources() {
+				if _, found := logsMetadata[logSource.Name]; !found {
+					logsMetadata[logSource.Name] = []metadata{}
+				}
+
+				parsedJSON, err := logSource.Config.PublicJSON()
+				if err != nil {
+					ic.log.Debugf("could not parse log configuration for source metadata %s: %v", logSource.Name, err)
+					continue
+				}
+
+				// Compute log source status
+				logSourceStatus := "pending"
+				if logSource.Status.IsSuccess() {
+					logSourceStatus = "success"
+				} else if logSource.Status.IsError() {
+					logSourceStatus = "error"
+				}
```

Comment:
The LogStatus has the `Dump()` function. We might be able to add something similar to that struct to get a human-readable string of the status, something like `logSource.Status.String()`

WDYT?